daemon/cmd/daemon_main.go

```diff
@@ -882,10 +882,6 @@ func initializeFlags() {
 	flags.Int(option.EndpointQueueSize, defaults.EndpointQueueSize, "Size of EventQueue per-endpoint")
 	option.BindEnv(Vp, option.EndpointQueueSize)
 
-	flags.Duration(option.EndpointGCInterval, 5*time.Minute, "Periodically monitor local endpoint health via link status on this interval and garbage collect them if they become unhealthy, set to 0 to disable")
-	flags.MarkHidden(option.EndpointGCInterval)
-	option.BindEnv(Vp, option.EndpointGCInterval)
```

Comment:
If possibly, maybe move the flag removal into a dedicated commit? I saw the note in the commit message about it, but it looked a bit odd/unrelated in the diff.